docker-compose.yml

```diff
@@ -1,7 +1,10 @@
-version: '3.4'
+version: '3.8'
 
 services:
   job:
     build:
       context: .
       dockerfile: ./dockerfiles/Dockerfile
+    env_file: .env
+    volumes:
+      - ${HOME}/secrets:/secrets
```

Comment:
Please download the google service account file from the dropbox and place in your secrets folder in home directory.